source/common/http/codes.cc

```diff
@@ -133,19 +154,63 @@ absl::string_view CodeStatsImpl::stripTrailingDot(absl::string_view str) {
   return str;
 }
 
-std::string CodeStatsImpl::join(const std::vector<absl::string_view>& v) {
-  if (v.empty()) {
-    return "";
+Stats::StatName CodeStatsImpl::upstreamRqGroup(Code response_code) const {
+  switch (enumToInt(response_code) / 100) {
+  case 1:
+    return upstream_rq_1xx_;
+  case 2:
+    return upstream_rq_2xx_;
+  case 3:
+    return upstream_rq_3xx_;
+  case 4:
+    return upstream_rq_4xx_;
+  case 5:
+    return upstream_rq_5xx_;
+  }
+  return upstream_rq_unknown_;
```

Comment:
I think I see why you added this. How was this handled before? I'm wondering if we need a new stat here.

Reply:
It was returning an empty string. Probably that functionality rarely saw the light of day, but we can bring back that behavior if we want ("foo.rq_" rather than "foo.rq_unknown"). I figured this would be easier to track down if it ever does happen. WDYT?

Reply:
Sure that's fine.